lib/hw_node.py

```diff
@@ -42,7 +59,7 @@ def init():
             network_manager.get_option('sls_list').split(',')]
 
 
-def get_ipmi_cycle_cmd(ip, user='', passwd=''):
+def  get_ipmi_cycle_cmd(ip, user='', passwd=''):
```

Comment:
I think `user=None, passwd=None` is better style, but I do not insist on this change.

Reply:
done